test/WebJobs.Script.Tests/FunctionsSyncServiceTests.cs

```diff
@@ -131,6 +131,15 @@ public void IsSyncTriggersEnvironment_StandbyMode_ReturnsExpectedResult(bool isA
         {
             _mockWebHostEnvironment.SetupGet(p => p.InStandbyMode).Returns(standbyMode);
 
+            if (isConsumptionLinuxOnLegion)
+            {
+                _mockEnvironment.Setup(p => p.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteSku)).Returns(ScriptConstants.FlexConsumptionSku);
```

Comment:
The sku checks for IsAnyLinuxConsumption are now accessing WEBSITE_SKU so we need to set the mock up